packages/components/src/components/data-grid/data-grid.tsx

```diff
@@ -493,6 +503,21 @@ export class DataGrid {
     this.activeSortingIndex = -1;
   }
 
+  presortIfNeeded(): void {
```

Comment:
Better call the fn `preSort` the if needed part is implied

Reply:
renamed and merged methods